### PR TITLE
Finalize message threading integration

### PR DESCRIPTION
**Pull request by HoonBaek**
## For Internal Contributors

[QU-289](https://sendbird.atlassian.net/browse/QU-289)
[QU-299](https://sendbird.atlassian.net/browse/QU-299)
[QU-309](https://sendbird.atlassian.net/browse/QU-309)
[QU-336](https://sendbird.atlassian.net/browse/QU-336)

## Description Of Changes

* Add channel event management logic for onThreadInfoUpdated
* Remove sendbird.min.js file and import released core SDK
* Fix condition about `customTypes` of message list params
* Cherry pick #47 
* 

## Types Of Changes

What types of changes does your code introduce to this project?
Put an `x` in the boxes that apply_

- [x] Bugfix
- [ ] New feature
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance (ex) Prettier)
- [ ] Build configuration
- [ ] Improvement (refactor code)

### CODE REVIEW

File: src/smart-components/ChannelList/utils.js [hunk 1]
```diff
@@ -248,7 +248,9 @@ export const pubSubHandler = (pubSub, channelListDispatcher) => {
   subScriber.set(topics.UPDATE_USER_MESSAGE, pubSub.subscribe(topics.UPDATE_USER_MESSAGE, (msg) => {
     const { channel, message } = msg;
     const updatedChannel = channel;
-    updatedChannel.lastMessage = message;
+    if (updatedChannel.lastMessage && updatedChannel.lastMessage.messageId === message.messageId) {
```
Comment (sravan-s) [inline src/smart-components/ChannelList/utils.js:251]: updatedChannel?.lastMessage?.messageId

Comment (HoonBaek) [reply src/smart-components/ChannelList/utils.js:251]: The type of this file is not typescript

Comment (sravan-s) [reply src/smart-components/ChannelList/utils.js:251]: its okay, works in js. Is JS feature

Comment (sravan-s) [reply src/smart-components/ChannelList/utils.js:251]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Comment (HoonBaek) [reply src/smart-components/ChannelList/utils.js:251]: Oh, I didn't know that. Thanks

File: src/smart-components/Conversation/dux/reducers.js [hunk 2]
```diff
@@ -214,7 +214,11 @@ export default function reducer(state, action) {
       };
     }
     case actionTypes.ON_MESSAGE_UPDATED: {
```
Comment (sravan-s) [inline src/smart-components/Conversation/dux/reducers.js:216]: Add test for these two changes

**Review by sravan-s — APPROVED** [review]
Couple of suggestions

File: src/smart-components/Conversation/dux/reducers.js (hunk 2 above)
Comment (HoonBaek) [reply src/smart-components/Conversation/dux/reducers.js:216]: There is already a test for `ON_MESSAGE_UPDATED` event.
Let me add new tests for `ON_MESSAGE_THREAD_INFO_UPDATED` event.

**Pull request merged**